src/Tasks/ResolveAssemblyReferences/Contract/IResolveAssemblyReferenceTaskHandler.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
should these files have the copyright header?

Reply:
Fixed